libcomps/src/python/src/pycomps.c

```diff
@@ -25,20 +25,10 @@
 
 #include <stdbool.h>
 
-#include "pycomps_23macros.h"
+#include "pycomps_macros.h"
 #include "pycomps.h"
 #include "pycomps_exc.h"
 
-#if PY_MAJOR_VERSION >= 3
-    #define MODINIT_RET_NONE return NULL
-    #define PY_OBJ_HEAD_INIT PyVarObject_HEAD_INIT(NULL, 0)
-    #define IS_PY3K
-#else
-    #define MODINIT_RET_NONE return
-    #define PY_OBJ_HEAD_INIT PyObject_HEAD_INIT(NULL)\
-                             0,
```

Comment:
All of these macros (and the ones in ```pycomps23_macros.h``` except for a few utility ones) I manually substituted since the abstraction is no longer necessary.